ts/backend/src/app/features/server/router.mts

```diff
@@ -351,15 +351,18 @@ export function router(_server: Server) {
         ${auth.sub ?? null},
         ${auth['preferred_username'] ?? null}
       )
-      RETURNING id
+      RETURNING id, uuid
     `
-    const id: number = idRow.at(0)?.['id'] ?? 0
+    const id: string = idRow.at(0)?.['id'] ?? 0
+    const uuid: string = idRow.at(0)?.['uuid'] ?? '00000000-0000-0000-0000-000000000000'
```

Comment:
Should we not check `id` and `uuid` are non-zero? And then fail with explicit exception. Better safe than sorry.